### PR TITLE
Actually fix undo resetting tools

### LINKED ISSUE #7: Undo feature undos tool selection
This bug/feature has been an ongoing discussion within the team.

### LINKED ISSUE #49: Menu item activation doesn't work with undo
When user actions are undone the currently selected tool changes depending on which tool was used last while the corresponding menu item stays unactivated.

### PR DESCRIPTION
**Pull request by ClFeSc**
Closes #49 
Closes #7 
This PR fixes an error made in #52

### CODE REVIEW

File: packages/SketchMorph2-Core.package/M2Backend.class/instance/undo.st [hunk 1]
```diff
@@ -1,6 +1,6 @@
 undo
 undo
-	self 
+	self wrapBlock: [self
```
Comment (kpostnov) [inline packages/SketchMorph2-Core.package/M2Backend.class/instance/undo.st]: If possible, `wrapBlock` should be renamed.

**Comment by kpostnov** [comment]
This PR should close #7 as well.

**Review by maximilian-franz — APPROVED** [review]
I agree with @ScrappyDieRaupe that `wrapBlock:` might not be the best name. Otherwise, I think this looks good.

**Review by DevSchmidtchen — APPROVED** [review]

**Review by johannes-poetzsch — APPROVED** [review]

**Comment by ClFeSc** [comment]
With the rename this now looks fine for me.

**Review by kpostnov — APPROVED** [review]

**Review by jannis-hpi — APPROVED** [review]

**Pull request merged**